EngineLayer/ClassicSearch/ClassicSearchEngine.cs

```diff
@@ -66,19 +74,22 @@ protected override MetaMorpheusEngineResults RunSpecific()
                 int[] threads = Enumerable.Range(0, maxThreadsPerFile).ToArray();
                 Parallel.ForEach(threads, (i) =>
                 {
-                    var fragmentsForDissociationTypes = new Dictionary<DissociationType, List<Product>>();
+                    var targetFragmentsForEachDissociationType = new Dictionary<DissociationType, List<Product>>();
+                    var decoyFragmentsForEachDissociationType = new Dictionary<DissociationType, List<Product>>();
```

Comment:
This is an unused variable?

Reply:
The decoyFragmentsForEachDissociationType? This is used in later part